Cargo.toml

```diff
@@ -244,8 +244,8 @@ opt-level = 0
 
 # Inherits by default from release
 [profile.bench]
-lto = false
-debug = true
+lto = "off"
```

Comment:
> false: Performs “thin local LTO” which performs “thin” LTO on the local crate only across its [codegen units](https://doc.rust-lang.org/cargo/reference/profiles.html#codegen-units).

https://doc.rust-lang.org/cargo/reference/profiles.html#lto

It is almost impossible for me to iterate otherwise due to compile times